spylon_kernel/init_spark_magic.py

```diff
@@ -37,8 +39,9 @@ def cell_init_spark(self):
 
         globals_dict = self.env
         exec(self.code, globals_dict)
+        application_name = globals_dict.get('application_name', None)
```

Comment:
Since I instantiate it as `None`, `get` is never actually going to return the default. Removing it for clarity.